cds_dojson/schemas/records/books/book/base-v0.0.1.json

```diff
@@ -162,10 +162,10 @@
               "default": "author",
```

Comment:
default should have uppercase as in the enum below

Reply:
well spotted! should be fixed.